File: poverty/src/poverty/County.java
package poverty;

public class County {
	
	public String name;
	public String povPercent;
	public String povCount;
	public String medIncome;

	public County(String line) {
		this.name = line.substring(193, 238);
		
		String[] details = line.split("\\s+");
		this.povPercent = details[11];
		this.povCount = details[8];
		this.medIncome = details[20];
	}

}

File: poverty/county.py
class County:
    NAME_START = 193
    NAME_END = 238
    def __init__(self, line):
        self.name = line[County.NAME_START:County.NAME_END]
        details = line.split()
        self.pov_percent = details[11]
        self.pov_count = details[8]
        self.med_income = details[20]
